x-pack/plugins/observability/public/pages/slos/components/slo_list_welcome_prompt.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiPageTemplate, EuiButton, EuiTitle, EuiLink, EuiImage } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import illustration from './assets/illustration.svg';
+
+export function SloListWelcomePrompt() {
+  return (
+    <EuiPageTemplate minHeight="0" data-test-subj="slosPageWelcomePrompt">
+      <EuiPageTemplate.EmptyPrompt
+        title={
+          <EuiTitle size="l">
+            <h1>
+              {i18n.translate('xpack.observability.slos.sloList.welcomePrompt.title', {
+                defaultMessage: 'Track and deliver on your SLOs',
+              })}
+            </h1>
+          </EuiTitle>
+        }
+        icon={<EuiImage size="fullWidth" src={illustration} alt="" />}
+        color="transparent"
+        layout="horizontal"
+        hasBorder={false}
+        body={
+          <>
+            <p>
+              {i18n.translate('xpack.observability.slos.sloList.welcomePrompt.messageParagraph1', {
+                defaultMessage:
+                  'Measure key metrics important to the business, such as service-level indicators and service-level objectives (SLIs/SLOs) to deliver on SLAs.',
```

Comment:
> Measure key metrics important to the business, such as service-level indicators and service-level objectives (SLIs/SLOs) to deliver on SLAs.

Did we run all these copies to product (Vinay, Tom) and ux-writing? We don't have to do it for this PR, but we should definitely create an issue to track.

Reply:
Agree. Created https://github.com/elastic/kibana/issues/147748